packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -170,9 +170,9 @@ contract FaultDisputeGame is Clone, ISemver {
     uint256 internal constant HEADER_BLOCK_NUMBER_INDEX = 8;
 
     /// @notice Semantic version.
-    /// @custom:semver 1.4.1
+    /// @custom:semver 1.4.2
     function version() public pure virtual returns (string memory) {
-        return "1.4.1";
+        return "1.4.2";
     }
```

Comment:
Why are we bumping this semver?

Reply:
It changes in the semver-lock for some reason, not sure why